appinfo/app.php

```diff
@@ -20,30 +20,8 @@
  *
  */
 
-use OCA\Notifications\App;
-use OCA\Notifications\Handler;
 use OCP\Util;
 
-\OC::$server->getNotificationManager()->registerApp(function() {
```

Comment:
why has this been removed?

Reply:
The registration has been moved to the Application constructor and it will use the new event-based way

Reply:
Ah - I see - THX